Log/Logger.php

```diff
@@ -40,7 +40,7 @@ public function log($level, $message, array $context = array())
                 $logMessage->setRequest(new LogRequest($context['request']));
             }
 
-            if (isset($context['response'])) {
+            if (isset($context['response']) && null != $context['response']) {
```

Comment:
using empty sounds better in this case (http://php.net/manual/en/function.empty.php) cause null values are also false.
But before changing that part we should investigate why response can be null.